sdk/python/tests/data/data_creator.py

```diff
@@ -57,8 +60,10 @@ def get_feature_values_for_dtype(dtype: str, is_list: bool) -> List:
         "bool": [True, None, False, True, False],
     }
     non_list_val = dtype_map[dtype]
-    # Duplicate the value once if this is a list
     if is_list:
+        if is_empty:
```

Comment:
can we add a test case of completely empty and asserting the error?

Reply:
The way the tests are set up that's a little tricky 😉 

Could add some unit tests to catch that?

Reply:
we can push this for later if you leave a TODO so we don't forget.

mind renaming `is_empty` to `has_empty_list` to be more precise?

Reply:
Done.